src/NzbDrone.Core/Indexers/Definitions/ImmortalSeed.cs

```diff
@@ -170,16 +170,21 @@ public ImmortalSeedRequestGenerator()
 
         private IEnumerable<IndexerRequest> GetPagedRequests(string term, int[] categories, string imdbId = null)
         {
-            var searchUrl = Settings.BaseUrl + "browse.php";
+            var searchUrl = Settings.BaseUrl + "browse.php?";
```

Comment:
What does this do if term is null and cats are null? does going to `https://..../browse.php?` actually work?

Reply:
per pearson yes; it returns all results
![image](https://user-images.githubusercontent.com/55419169/152662748-72de9135-4930-4139-acbd-688534719adc.png)